src/fides/ctl/core/config/__init__.py

```diff
@@ -31,16 +31,23 @@
 class FidesConfig(BaseModel):
     """Umbrella class that encapsulates all of the config subsections."""
 
+    # Pydantic doesn't initialise subsections automatically if
```

Comment:
My gut tells me there's a better way to solve this.

Additionally, this functionality is now degrading the user experience. We somehow need to track required key-value pairs and prompt the user for them, or at least nicely print out "hey, these are required and the app can't start without them.", give some nice hand-holding so that users aren't immediately hit with `key not found` errors when trying to start up the application

Reply:
I don’t like this solution at all, but it is the only way we have found to do it. Sean and I found this solution looking through Pydantic issues when we were working on fidesops. The issue is with nested Pydantic models, it tries to populate them without the environment variables and config file first so it errors. 

If you have thoughts as to another way it would be great. I think it is the root of all the issues here. Sean and I spent a few hours looking for other solutions initially, and I spent another few hours yesterday with no luck.

Reply:
why not load each module directly instead of counting on clever nested models?

i.e.

```

Reply:
I'll give it a go.

Reply:
OK, to summarize this conversation- we've not found a solution we like (yet), so we reverted most of those changes for now. But it does appear we can use this style for just one model: the `SecuritySettings`. Is that accurate?